src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -342,4 +342,16 @@
   <data name="PX1054Title" xml:space="preserve">
     <value>A PXGraph instance should not start PXLongOperation during the initialization phase</value>
   </data>
+  <data name="PX1055Fix1" xml:space="preserve">
+    <value>Make the field bound to identity column the only key field in the DAC</value>
+  </data>
+  <data name="PX1055Fix2" xml:space="preserve">
+    <value>Alter the definition of the field bound to identity column to make it a non-key field and leave all other key fields untouched</value>
```

Comment:
@k-s-popova , please verify string

Reply:
@saloponov , please change as follows:
Make the field bound to the identity column be a non-key field and leave all other key fields untouched